api/src/apolloClient.ts

```diff
@@ -143,6 +147,9 @@ export const initApolloServer = async (app) => {
             webSocket.messengerData = parsedMessage.value;
 
             const customerId = webSocket.messengerData.customerId;
```

Comment:
Can webSocket or webSocket.messengerData be undefined or null? If so, this line could throw error.